inc/class-wpseo-options.php

```diff
@@ -2791,192 +2791,195 @@ protected function clean_option( $option_value, $current_version = null, $all_ol
 /*******************************************************************
  * Option: wpseo_ms
  *******************************************************************/
-if ( is_multisite() ) {
```

Comment:
I don't have the time at the moment to look at it and argue it in detail. The short of it is that your are changing the expected behaviour of the object without cause.

I suggest you either change the conditional in the `WPSEO_Options` class as I suggested before or add the conditional to the `get_instance()` method of the _ms class instead of the constructor. That way the behaviour doesn't change (instance will be null rather than a nearly empty object).
The constructor of the `WPSEO_Options` class needs to change either way as the whole `class_exists()` doesn't make sense anymore with all option classes always being available.